model-server/src/main/kotlin/org/modelix/model/server/handlers/ModelReplicationServer.kt

```diff
@@ -170,7 +170,7 @@ class ModelReplicationServer(
             fun ApplicationCall.repositoryId() = RepositoryId(parameters["repository"]!!)
             fun PipelineContext<Unit, ApplicationCall>.repositoryId() = call.repositoryId()
 
-            val useRoleIds = call.request.queryParameters["useRoleIds"] != "false"
+            val useRoleIds = call.request.queryParameters["useRoleIds"]?.toBooleanStrict() != false
```

Comment:
I'd rather not change the behavior because some code may rely on it.
See https://www.hyrumslaw.com/

I'm definitely for such strict checks and would strongly prefer them in new endpoints.

The change in the definition is good.

Reply:
`?useRoleIds=False` is interpreted as `true`... I'm for making it strict now.